scripts/crawl.py

```diff
@@ -0,0 +1,86 @@
+import bs4
+import json
+import requests
+from pathlib import Path
+from typing import Union
+from argparse import ArgumentParser
+
+
+def parse_arguments():
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument('--input_dir', '-d', default='/home/elte-dh-clymene/dh/proba', type=Path,
+                        help='Directory where previous info is stored.')
+    parser.add_argument('--url', '-u', type=str, default="https://horvlu.web.elte.hu/",
+                        help='REST API url')
+    args = parser.parse_args()
+    return args
+
+
+months_dict = {
+    'January': '01',
+    'February': '02',
+    'March': '03',
+    'April': '04',
+    'May': '05',
+    'June': '06',
+    'July': '07',
+    'August': '08',
+    'September': '09',
+    'October': '10',
+    'November': '11',
+    'December': '12',
+    'Winter': '12',
+    'Summer': '06',
+    'Spring': '03',
+    'Fall': '09',
+}
+
+
+def get_info_from_line(li: bs4.element.Tag) -> Union[tuple, None]:
+    if "CC-MAIN" in li.text:
```

Comment:
Please add docstrings to all functions, shortly describing what they do. Refer to one of the other `.py` files in the repository to see how to format them.